File: src/IsbInterop/Data/IPickRequisiteItem.vb
Namespace Data

  ''' <summary>
  ''' Описание допустимого значения реквизита типа «Признак».
  ''' </summary>
  Public Interface IPickRequisiteItem
    Inherits IBaseIsbObject

    ''' <summary>
    ''' Внутренний ИД допустимого значения реквизита, хранящийся в базе данных.
    ''' </summary>
    ReadOnly Property Id As Integer
  End Interface
End NameSpace

File: src/IsbInterop/Data/Requisites/IPickRequisiteItem.cs
namespace IsbInterop.Data.Requisites
{
  /// <summary>
  /// Описание допустимого значения реквизита типа «Признак».
  /// </summary>
  public interface IPickRequisiteItem : IBaseIsbObject
  {
    /// <summary>
    /// Внутренний ИД допустимого значения реквизита, хранящийся в базе данных.
    /// </summary>
    int Id { get; }
  }
}
